### DISCUSSION TITLE
我是完全的萌新

### DISCUSSION THREAD

**Discussion by gudaoxifengshouma** [ByteLegend/ByteLegend#43] [General]
我想知道留名字的那个代码下面的propose那个键是灰色的，点不动是为什么

**Comment by blindpirate** [comment]
没有截图，我们恐怕无法帮忙。不过按照我的经验，通常是因为你没有做出任何更改，所以无法`Propose change`。

**Reply by gudaoxifengshouma** [reply]
抱歉给您带来困扰，当时由于图片无法显示，才会有一系列的问题，在昨天图片能正常显示了，我已经弄好了，抱歉让您费心了，祝您生活愉快

**Reply by blindpirate** [reply]
:tada: 无需抱歉，每个人都曾经是新手。未来有机会的话，请尽你所能帮助其他人。
